arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java

```diff
@@ -68,10 +76,58 @@ public <T extends Mocker> boolean putRecordResult(MockCategoryType category, Str
                 }
             }
         }
+        if (shouldUseIdOfInstanceToMockResult(category) && CollectionUtils.isNotEmpty(unusedRecordInstanceList)) {
+            putRecordInstanceData(category, recordIdBytes, unusedRecordInstanceList);
+        }
         LOGGER.info("put record result to cache size:{} for category:{},record id:{}", size, category, recordId);
         return size > EMPTY_SIZE;
     }
 
+    /**
+     * put RecordInstanceData to cache
+     */
+    private void putRecordInstanceData(MockCategoryType category, byte[] recordIdBytes, List<RecordInstanceData> unusedRecordInstanceList) {
+        byte[] unusedRecordInstanceKey = getRecordInstanceDataKey(category, recordIdBytes);
+        byte[] value = null;
+        try {
+            value = serializer.serialize(objectMapper.writeValueAsString(unusedRecordInstanceList));
```

Comment:
> value = serializer.serialize(objectMapper.writeValueAsString(unusedRecordInstanceList));

Why use objectMapper serialization instead of ZstdJacksonSerializer directly?